torch_xla/_internal/rendezvous.py

```diff
@@ -39,10 +39,12 @@ def pjrt_rendezvous_handler(url: str,
     global _store
     if not _store:
       # Create DummyStore when user disables TORCH_DIST_INIT_BARRIER
-      # to skip store based barriers. It's safe to do this because store
-      # created by _pjrt_rendezvous_handler is only used as a barrier in
-      # process group initialization.
-      if xu.getenv_as('TORCH_DIST_INIT_BARRIER', int, 1) == 0:
+      # to skip store based barriers. It's safe to do this in Neuron NeMo Megatron
+      # because store created by _pjrt_rendezvous_handler is only used as a barrier
+      # process group initialization. In NxD, the store is also used to exchange python
+      # objects. So, to use DummyStore, set TORCH_DIST_INIT_BARRIER=0 and NEURON_USE_DUMMY_STORE=1
+      if xu.getenv_as('TORCH_DIST_INIT_BARRIER', int, 1) == 0 and xu.getenv_as(
+          'NEURON_USE_DUMMY_STORE', int, 1) == 1:
```

Comment:
Can this be changed to be more generic? This PR should go to mainline also, so having something that work across platform should be better.

Reply:
Sure, will change this environment variable to `TORCH_USE_DUMMY_STORE`

Reply:
Most of our env var options are prefixed with `XLA_`. Can you make this `XLA_USE_DUMMY_STORE` instead?

Reply:
Sure